### DISCUSSION TITLE
[Bug]: Link should be rendered similarly to a button in Datatable > Toolbar setup

### DISCUSSION THREAD

**Discussion by Pierstoval** [carbon-design-system/carbon-components-svelte#1916] [Q&A] [answered]
When using the DataTable component and the <Toolbar> default setup, we can use <Button> component to add an element, just like in the [examples in the docs](https://carbon-components-svelte.onrender.com/components/DataTable#with-toolbar). 

However, <Button> elements are only there for JS-based interaction, and should not really open links.

When you want to actually have a real link pointing to an URL, the <Link> element is not rendered with proper styling:

![image](https://github.com/carbon-design-system/carbon/assets/3369266/543fda2f-3723-4251-adb4-f3b0432edd8e)

Where can I go to fix this?

Should I create a ticket on the `carbon-design-system/carbon` repo?

**Comment by brunnerh** [comment, marked as answer]
Would just use a `Button`, [they can act as links](https://carbon-components-svelte.onrender.com/components/Button#link-button) (they have a `href` property).
Set it to `kind="ghost"` for less prominent look.
